packages/flutter/test/material/reorderable_list_test.dart

```diff
@@ -541,6 +545,97 @@ void main() {
           ));
           handle.dispose();
         });
+
+        testWidgets('ReorderableListView still scrolls when physics property is set', (WidgetTester tester) async {
+          final List<Widget> widgetList = <Widget>[
+            Container(key: const Key('1'), child:  const Icon(Icons.account_circle))
+          ];
+          final SingleChildScrollView scrollView = SingleChildScrollView(
+            child: SizedBox(
+              height: 100,
+              width: 100,
+              child: ReorderableListView(
+                key: const Key('list'),
+                physics: const AlwaysScrollableScrollPhysics(),
+                onReorder: (int oldIndex, int newIndex) {},
+                children: widgetList,
+                scrollController: ScrollController(),
+              )
+            ),
+          );
+          await tester.pumpWidget(MaterialApp(
+          home: SizedBox(
+            height: 100,
+            width: 100,
+            child: scrollView,
+          ),
+        ));
+
+          final TestGesture gesture = await tester.startGesture(tester.getCenter(find.byType(ReorderableListView)));
+          await gesture.moveBy(_kGestureOffset);
+          // Move back to original position.
+          await gesture.moveBy(Offset(-_kGestureOffset.dx, -_kGestureOffset.dy));
+          await tester.pump();
+          await tester.pump(const Duration(milliseconds: 500));
+
+          final TestGesture overscrollGesture = await tester.startGesture(tester.getCenter(find.byType(ReorderableListView)));
+          await overscrollGesture.moveBy(_kGestureOffset);
+          await tester.pump();
+          await tester.pump(const Duration(milliseconds: 500));
+
+          expect(tester.takeException(), isNull);
```

Comment:
Wouldn't it be better to, instead of testing for the lack of an exception, to test whether or not the scroll behavior is present?

Reply:
Updated